inc/project.class.php

```diff
@@ -1827,135 +1827,74 @@ static function getDataToDisplayOnGantt($ID, $showall = true) {
       return $todisplay;
    }
 
-
-   /** show GANTT diagram for a project or for all
-    *
-   * @param $ID ID of the project or -1 for all projects
+   /**
+     * Show GANTT diagram for a project
+     * @param $ID ID of the project
    */
    static function showGantt($ID) {
-      global $DB;
+      echo "<div id=\"gantt-container\" style=\"width:100%; height:63vh;\"></div>";
+      echo "<div id=\"gantt-features\">
+               <ul class=\"gantt-controls\">
+                  <li class=\"gantt-menu-item gantt-menu-item-right\">
+                  <a href=\"#\" class=\"fullscreen\" onclick=\"gantt.ext.fullscreen.toggle();\"><img src=\"../pics/icones/ic_fullscreen_24.png\">Fullscreen</a>
```

Comment:
Should certainly be replaced with a fontawesome icon

Reply:
it's done